homeassistant/components/netatmo.py

```diff
@@ -50,7 +49,4 @@ def setup(hass, config):
         _LOGGER.error("Unable to connect to NatAtmo API")
         return False
 
-    for component in 'camera', 'sensor':
```

Comment:
No the camera component can setup as many camera as you want, but the thing is that might don't want to have all the information from the camera, for instance, the binary sensor create 3 sensor by camera by default, bu user might don't want to have the 'Someone known' sensor if they have a more precise device tracker (moreover with my next update which should implement presence detection based on the Netatmo camera).

I would me more than happy to use the discovery, but discovery use the default config (all camera, all sensors) but if I setup a specific config it creates duplicate in the interface (the feed from discovery and the one from specific setup)